src/components/custom-field-input-date/custom-field-input-date.jsx

```diff
@@ -59,6 +59,14 @@ const CustomFieldInputDate = forwardRef(function CustomFieldInputDate(props, ref
     setShouldFocusInput(false);
   }, [shouldFocusInput]);
 
+  useEffect(() => {
+    if (ref && currentValue) {
+      props.onChange({ target: ref.current });
```

Comment:
I think we can get coverage on the parameters for `onChange`. At the moment, it looks like the tests only check that it was called. We know that we need this specific signature to make it work with Form.